SingularityUI/app/models/TaskResourceUsage.coffee

```diff
@@ -6,4 +6,19 @@ class TaskResourceUsage extends Model
 
     initialize: ({ @taskId }) =>
 
-module.exports = TaskResourceUsage
+    # Calculate CPU Usage by comparing previous usage to current usage
+    setCpuUsage: ->
+        previous = @get('previousUsage')
+        currentTime = @get('cpusSystemTimeSecs') + @get('cpusUserTimeSecs')
+        previousTime = previous.cpusSystemTimeSecs + previous.cpusUserTimeSecs
+        timestampDiff = @get('timestamp') - previous.timestamp
+        cpus_used_raw = (currentTime - previousTime) / timestampDiff
+        cpus_used = Math.round(cpus_used_raw * 100) / 100        
```

Comment:
might be best if we pass the full precision value through, and then use `.toFixed(2)` when rendering